tests/test_api.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Test the Quantity class Array API compatibility."""
+
+import astropy.units as u
+import numpy as np
+
+from quantity import Quantity, api
+
+
+def test_issubclass_api():
+    """Test that Quantity is a subclass of api.Quantity and api.QuantityArray."""
+    assert issubclass(Quantity, api.Quantity)
+    assert issubclass(Quantity, api.QuantityArray)
```

Comment:
How about we also check `u.Quantity`? It should work and we probably want it to continue to work.

Maybe also good to do `assert not issubclass(np.ndarray, ...`

Reply:
Done.